pandas/tests/test_graphics.py

```diff
@@ -1065,12 +1065,6 @@ def test_implicit_label(self):
         self._check_text_labels(ax.xaxis.get_label(), 'a')
 
     @slow
-    def test_explicit_label(self):
```

Comment:
Hey @TomAugspurger - this test checks to see if th axis is relabeled and not if there is a legend title label. Per the issue discussion we thought the axis shouldn't be changed, but just the series name within the legend. There is another test that checks legend that I edited but can re-edit